source/materials/MaterialsList.cc

```diff
@@ -486,11 +488,13 @@ G4Material* MaterialsList::ITO()
     G4NistManager* nist = G4NistManager::Instance();
 
     G4Element* In = nist->FindOrBuildElement("In");
-    G4Element* O = nist->FindOrBuildElement("O");
+    G4Element* O  = nist->FindOrBuildElement("O");
+    G4Element* Sn = nist->FindOrBuildElement("Sn");
 
-    mat = new G4Material(name, 7.14*g/cm3, 2, kStateSolid);
+    mat = new G4Material(name, 6.8*g/cm3, 3, kStateSolid);
     mat->AddElement(In, 2);
```

Comment:
Where are these number coming from? I can't find them in the link you provide. In any case, everywhere else the composition is given by weight.

Reply:
In the Wikipedia I found 


_Indium tin oxide is typically encountered as an oxygen-saturated composition with a formulation of 74% In, 18% O2, and 8% Sn by weight._


 (https://en.wikipedia.org/wiki/Indium_tin_oxide)

Reply:
Here is another, specific, ITO by Kurt J. Lesker: https://www.lesker.com/newweb/deposition_materials/depositionmaterials_sputtertargets_1.cfm?pgid=in3.
I'm not sure about the way we should implement it.

Reply:
The density comes from the commented link.
Respect the composition of the materials, in nexus some of them are given by weight, and many others by the number of atoms (TPB, HDPE, OpticalSilicone, Kevlar, PVT ...).
But if you prefer, I can change it to compositions by weight.

Reply:
> Respect the composition of the materials, in nexus some of them are given by weight, and many others by the number of atoms (TPB, HDPE, OpticalSilicone, Kevlar, PVT ...). But if you prefer, I can change it to compositions by weight.

But where did you get the numbers for the composition by number of atoms? I can't find them in that website. I think I'd use the composition by weight in this particular case, as it appears to have varying proportions.

Reply:
Done